projects/demo/src/utils/load-assets.ts

```diff
@@ -1,6 +1,3 @@
 export function assets(path: TemplateStringsArray): string {
-    const dirname = new URL(`.`, import.meta.url);
-    const relativePath = `../assets${path.join(``)}`;
-
-    return new URL(relativePath, dirname).toString();
+    return `./assets/${path.join(``)}`;
```

Comment:
What's up with this change?

Reply:
for some reason the previous version returned paths like `file:///Users/.../taiga-ui/projects/demo/src/assets/images/...` which caused the error `Not allowed to upload local resource`